Types of things like `Document`, etc, seem not exported in v7

I use the API of p5 to get certain nodes. I then transform those nodes. For that, I need their types. In v7, it seems that they are not exported, compared to `@types/parse5`.
Am I missing something? If not, is there a good reason they’re not exported? If not, can they be added?

I think it’s fair to say that other folks will want to walk and change the tree, so hence I’m for exposing them from a more visible place (and as part of semver).

I’m also finding that generating some nodes can be tough:

```
lib/index.js:63:5 - error TS2322: Type '"#document"' is not assignable to type 'NodeType.Document'.

63     nodeName: '#document',
       ~~~~~~~~

  node_modules/parse5/dist/tree-adapters/default.d.ts:13:5
    13     nodeName: NodeType.Document;
           ~~~~~~~~
    The expected type comes from property 'nodeName' which is declared here on type 'Document'

lib/index.js:64:5 - error TS2322: Type '"quirks" | "no-quirks"' is not assignable to type 'DOCUMENT_MODE'.
  Type '"quirks"' is not assignable to type 'DOCUMENT_MODE'.

64     mode: (node.data || {}).quirksMode ? 'quirks' : 'no-quirks',
       ~~~~

  node_modules/parse5/dist/tree-adapters/default.d.ts:18:5
    18     mode: DOCUMENT_MODE;
           ~~~~
    The expected type comes from property 'mode' which is declared here on type 'Document'
```

As far as I’m aware, such things should be fine?

--- a/packages/parse5/lib/index.ts
+++ b/packages/parse5/lib/index.ts
@@ -7,7 +7,7 @@ export { type DefaultTreeAdapterMap, defaultTreeAdapter } from './tree-adapters/
 export type { TreeAdapter, TreeAdapterTypeMap } from './tree-adapters/interface.js';
 export { type ParserOptions, /** @internal */ Parser } from './parser/index.js';
 export { serialize, serializeOuter, type SerializerOptions } from './serializer/index.js';
-export type { ParserError } from './common/error-codes.js';
+export { ERR as ErrorCodes, type ParserError } from './common/error-codes.js';
 
 /** @internal */
 export * as foreignContent from './common/foreign-content.js';

